pkg/config/lpe_unix.go

```diff
@@ -0,0 +1,56 @@
+//go:build aix || darwin || dragonfly || freebsd || netbsd || openbsd || solaris || zos
```

Comment:
This might need `|| linux`?

Reply:
Removed them. I'm still trying to make a better understanding of file suffixes, build tags and their differences by go version and how everything works together. At least `make` was successful for all targets locally.